plugin/manifest.yaml

```diff
@@ -104,23 +104,6 @@ data:
     }
 ---
 apiVersion: v1
-kind: ConfigMap
-metadata:
-  name: plugin-conf
-  namespace: ossmconsole
-  labels:
-    app: ossmconsole
-    app.kubernetes.io/component: ossmconsole
-    app.kubernetes.io/instance: ossmconsole
-    app.kubernetes.io/name: ossmconsole
-    app.kubernetes.io/part-of: ossmconsole
-data:
-  plugin-config.json: |
-    {
-      "kialiUrl": "https://kiali-istio-system.apps-crc.testing"
```

Comment:
I see the `kialiUrl` is being removed in a bunch of places in this PR.

Is `kialiUrl` going to go away completely? If so, a bunch of changes need to be made to the operator ([here](https://github.com/kiali/openshift-servicemesh-plugin/blob/main/operator/roles/default/ossmconsole-deploy/templates/openshift/configmap-plugin.yaml#L10) and related places)

Reply:
Yes, kialiUrl is not used since there are no iframes to redirect to kialiUrl. I will try to change as much as I can but probably I wll need your help on operator @jmazzitelli

Reply:
BTW: if we get rid of the kialiUrl, how does the plugin know which backend Kiali server to talk to?

Reply:
kialiUrl is used just for iframes, backend kiali service is set in OSSMConsole CRD:

https://github.com/kiali/openshift-servicemesh-plugin/blob/057308b64bb57a9f1a11fb193fa87eb2802891ed/operator/crd-docs/crd/kiali.io_ossmconsoles.yaml#L85

Reply:
Oh.... I see it... its the proxy definition in the ConsolePlugin object: https://github.com/ferhoyos/openshift-servicemesh-plugin/blob/main/operator/roles/default/ossmconsole-deploy/templates/openshift/consoleplugin.yaml#L13-L20